package.json

```diff
@@ -13,7 +13,7 @@
   ],
   "scripts": {
     "build": "turbo run build build:css --filter='./packages/@uppy/*' --filter='./packages/uppy'",
-    "build:clean": "cp .gitignore .gitignore.bak && printf '!node_modules\n!**/node_modules/**/*\n' >> .gitignore; git clean -Xfd packages e2e .parcel-cache coverage .turbo; mv .gitignore.bak .gitignore",
+    "build:clean": "git clean -Xfd packages e2e .parcel-cache coverage .turbo -e '!node_modules' -e '!**/node_modules/**/*'",
```

Comment:
I seriously doubt this works the same. Have you tested if I removes all nested node_modules? Have you tested running this command while you have uncommited changes?

Reply:
Yes :) it keeps all unstaged, staged, and untracked local changes. it deletes everything from .gitignore but excludes node_modules (which we want)